### PR TITLE
Fix non-netcore warning about add_assembly_to_alc.

### PR DESCRIPTION
**Pull request by jaykrell**

### CODE REVIEW

**Comment by jaykrell** [comment]
@monojenkins build failed

File: mono/mini/aot-compiler.c [hunk 1]
```diff
@@ -13336,10 +13336,6 @@ mono_compile_assembly (MonoAssembly *ass, guint32 opts, const char *aot_options,
 		aot_printerrf (acfg, "--aot=llvmonly requires a runtime compiled with llvm support.\n");
 		return 1;
 	}
-	if (mono_use_llvm || acfg->aot_opts.llvm) {
```
Comment (vargaz) [inline mono/mini/aot-compiler.c]: Unrelated.

Comment (jaykrell) [reply mono/mini/aot-compiler.c]: Incorrect merge, will undo.

Comment (jaykrell) [reply mono/mini/aot-compiler.c]: Incorrect merge, will undo.

**Comment by CoffeeFlux** [comment]
@monojenkins build failed
